app/src/main/java/com/blockeq/stellarwallet/activities/RecoverWalletActivity.kt

```diff
@@ -84,6 +91,30 @@ class RecoverWalletActivity : BaseActivity() {
             })
             builder.show()
         }
+
+        phraseEditText.addTextChangedListener(object : TextWatcher {
```

Comment:
let's remove all methods not used creating an abstract class with empty implementations called `onTextChanged`